src/components/Reviews.jsx

```diff
@@ -1,27 +1,32 @@
 import { getReviews } from "../api"
 import { useEffect } from "react"
 import { useState } from "react"
-import { Link, useParams } from "react-router-dom"
+import { Link, useParams, useSearchParams } from "react-router-dom"
+import moment from "moment"
 
 const Reviews = () => {
 
     const [isLoading, setIsLoading] = useState(true)
     const [reviews, setReviews] = useState([])
     const [error, setError] = useState(null)
     const { slug } = useParams()
+    const [sortBy, setSortBy] = useState('created_at')
+    const [orderBy, setOrderBy] = useState('desc')
+    const [searchParams, setSearchParams] = useSearchParams()
 
     useEffect(() => {
         setError(null)
         setIsLoading(true)
-        getReviews(slug)
+        getReviews(slug, sortBy, orderBy)
         .then((reviewsFromApi) => {
             setReviews(reviewsFromApi)
+            setSearchParams({"sort_by": sortBy, "order": orderBy})
```

Comment:
Great use of setSearchParams to ensure these are updated in the url when the dropdown menus change 👍